OpenSim/Simulation/Model/Model.h

```diff
@@ -220,18 +220,23 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Model, ModelComponent);
     //--------------------------------------------------------------------------
 public:
 
-    /** Default constructor creates a %Model containing only the ground Body
+    /** Default constructor creates a %Model containing only the Ground frame
     and a set of default properties. */
     Model();
 
     /** Constructor from an OpenSim XML model file. 
+    NOTE: The Model is read in (deserialized) from the model file, which means
+    the properties of the Model and its components are filled in from values in
+    the file. In order to evaluate the validity of the properties (e.g. Inertia
+    tensors, availability of Mesh files, and to traverse the tree of the
+    Model's subcomponents) one must invoke Model::finalizeFromProperties() 
+    first. Model::initSystem() invokes finalizeFromProperties() on its way to
+    creating the System and initializing the State.
```

Comment:
This comment is very good but I still see a small bit of room for confusion, since there is a difference between not having any subcomponents and those subcomponents not yet being "registered." Consider the following change:

"and to traverse the tree of the Model's subcomponents)" -> "to register all components with the Model"

and change the `ComponentIsRootWithNoSubcomponents` message as so:

"is the root but has no subcomponents. Verify that finalizeFromProperties() was called on this Component" ->
"is the root but no subcomponents are registered. Verify that subcomponents are registered by calling finalizeFromProperties() on this Component".